docs/ref/contrib/gis/functions.txt

```diff
@@ -261,13 +261,16 @@ right-hand rule.
 ``GeoHash``
 ===========
 
-.. class:: GeoHash(expression, **extra)
+.. class:: GeoHash(expression, precision=None, **extra)
 
 *Availability*: PostGIS, SpatiaLite (LWGEOM)
 
 Accepts a single geographic field or expression and returns a `GeoHash`__
 representation of the geometry.
 
+The number of returned characters is controlled by the ``precision`` keyword
```

Comment:
First sentence looks good. I think we could omit the second sentence as that's probably true for most of these functions.